cmd/ocm-backplane/cloud/ssm.go

```diff
@@ -209,8 +225,64 @@ func startSSMsession(cmd *cobra.Command, argv []string) error {
 		return fmt.Errorf("failed to get instance ID for node %s: %w", ssmArgs.node, err)
 	}
 
-	logger.Infof("Starting SSM session for node: %s with Instance ID: %s", ssmArgs.node, instanceID)
+	logger.Infof("Non-interactive command to be executed: %s", strings.Join(execCommand, " "))
+	if execCommand != nil {
+		logger.Infof("Starting SSM session for node: %s with Instance ID: %s to execute command: %s",
+			ssmArgs.node, instanceID, strings.Join(execCommand, " "))
+		return startSSMSessionWithCommand(ssmClient, instanceID, execCommand, creds.Region)
+	} else {
+		logger.Infof("Starting interactive SSM session for node: %s with Instance ID: %s", ssmArgs.node, instanceID)
+		return startSSMSessionRegular(ssmClient, instanceID, creds.Region)
+	}
+}
+
+// Start an SSM session to execute the provided command non-interactively
+func startSSMSessionWithCommand(ssmClient SSMClient, instanceID string, command []string, region string) error {
```

Comment:
It is flexible to put the common logic of `startSSMSessionWithCommand` and `startSSMSessionRegular` in a base function? We can maintain less if we can do so.